libs/cart/src/actions/cart.actions.ts

```diff
@@ -32,46 +33,66 @@ export class DaffCartLoadFailure implements Action {
   constructor(public payload: string) {}
 }
 
+export class DaffCartCreate implements Action {
+  readonly type = DaffCartActionTypes.CartCreateAction;
+}
+
+export class DaffCartCreateSuccess<T extends DaffCart> implements Action {
+  readonly type = DaffCartActionTypes.CartCreateSuccessAction;
+
+  constructor(public payload: {id: T['id']}) {}
+}
+
+export class DaffCartCreateFailure implements Action {
+  readonly type = DaffCartActionTypes.CartCreateFailureAction;
+
+  constructor(public payload: string) {}
+}
+
 export class DaffAddToCart implements Action {
-    readonly type = DaffCartActionTypes.AddToCartAction;
+  readonly type = DaffCartActionTypes.AddToCartAction;
 
-    constructor(public payload: {productId: string, qty: number}) {}
+  constructor(public payload: {productId: string, qty: number}) {}
```

Comment:
seems inconsistent that the `DaffCartCreateSuccess` action takes a payload of `{id: T['id']}` while this takes a payload of `{productId: string, qty: number}`. Seems like it should be something like `{productId: T['id'], qty: T['quantity']}`

Reply:
You're right, it is inconsistent. I didn't update anything related to the deprecated `addToCart` call.

Reply:
oh, right. I forgot it was deprecated. Nevermind then.